en/simulation/README.md

```diff
@@ -159,32 +159,64 @@ The simulated camera is a gazebo plugin that implements the [MAVLink Camera Prot
+
+
+### SSH Tunneling
+
+SSH tunneling is a flexible option because the simulation computer and the system using it need not be on the same network.
+
+> **Note** You might similarly use VPN to provide a tunnel to an external interface (on the same network or another network).
+
+One way to create the tunnel is to use SSH tunneling options.
+The tunnel itself can be created by running the following command on *localhost*, where `remote.local` is the name of a remote computer:
 ```
 ssh -C -fR 14551:localhost:14551 remote.local
-```
-Where "remote.local" is the name of a remote computer. 
+``` 
 
-Unfortunately, the SSH itself cannot route the UDP packets.  Therefore the UDP packets need to be translated to TCP packets. By [netcat](https://en.wikipedia.org/wiki/Netcat) utility separately on the local and remote side of the tunnel. 
-Local side of UDP packet translation of QGC could be implemented by running following commands.
+The UDP packets need to be translated to TCP packets so they can be routed over SSH. 
+The [netcat](https://en.wikipedia.org/wiki/Netcat) utility can be used on both sides of the tunnel - first to convert packets from UDP to TCP, and then back to UDP at the other end.
+
+> **Tip** QGC must be running before executing *netcat*.
+ 
+On the *QGroundControl* computer, UDP packet translation may be implemented by running following commands:
 ```
 mkfifo /tmp/tcp2udp
 netcat -lvp 14551 < /tmp/tcp2udp | netcat -u localhost 14550 > /tmp/tcp2udp
 ```
-For the remote side of the tunnel, the command differs. 
+On the simulator side of the SSH tunnel, the command is: 
 ```
 mkfifo /tmp/udp2tcp
 netcat -lvup 14550 < /tmp/udp2tcp | netcat localhost 14551 > /tmp/udp2tcp
 ```
-It is necessary to have QGC running before executing the netcat. 
-The tunnel could run infinitely, but netcat connections may need a restart in case of improper communication state occurs. The port number `14550` is valid for QGC software connection and should be adjusted for other possible communication channels.  
 
-The automated [bash connection script](https://raw.githubusercontent.com/ThunderFly-aerospace/sitl_gazebo/autogyro-sitl/scripts/QGC_remote_connect.bash) is prepared for automation of QGC to simulation server running the PX4 stack. 
+The port number `14550` is valid for connecting to QGroundControl or another GCS, but should be adjusted for other endpoints (e.g. developer APIs etc.).  
+
+The tunnel may in theory run indefinitely, but *netcat* connections may need to be restarted if there is a problem.
 
+The automated [bash connection script](https://raw.githubusercontent.com/ThunderFly-aerospace/sitl_gazebo/autogyro-sitl/scripts/QGC_remote_connect.bash) can be used to automate setup of QGC to simulation server running the PX4 stack. 
```

Comment:
This script just sets up the QGC side right? I haven't looked at it :-) What about the other side (SITL)? 
Ie if I have this script, how does process change? Does it install netcat and SSH etc?

Reply:
It does the other side as well. It assumes that you can `ssh` into the remote machine (generally given for a server). The simulation also must be running already.

Reply:
The script establishes both sides of the tunnel.  The script assumes a running simulation (SITL on the server) and QGC on local.

Reply:
Thanks. Fixed.